packages/manager/apps/hub/src/index.scss

```diff
@@ -0,0 +1,4 @@
+@import '~bootstrap4/scss/_functions';
```

Comment:
If custom rules need to be added, it  might be easier if the imports were "scoped" I would say.

Reply:
Custom rules will need to be scoped by components I agree, but here does it make sense to scope to `body` ? There is no component yet and I think we'll need the bootstrap grid for the app

Reply:
Looks good to me.